index.html

```diff
@@ -207,56 +210,70 @@ <h1 class="title">
   <td><a href="https://c.delta.chat">c.delta.chat</a></td>
   <td><a href="https://github.com/deltachat/deltachat-core-rust/tree/master/deltachat-ffi">/deltachat-ffi</a></td>
 </tr>
+<tr>
+  <td>JSON-RPC</td>
+  <td></td>
+  <td><a href="https://github.com/deltachat/deltachat-core-rust/tree/master/deltachat-jsonrpc">/deltachat-jsonrpc</a></td>
+</tr>
 <tr>
   <td>Python</td>
   <td><a href="https://py.delta.chat">py.delta.chat</a></td>
   <td><a href="https://github.com/deltachat/deltachat-core-rust/tree/master/python">/python</a></td>
 </tr>
 <tr>
-  <td>JSON-RPC</td>
+  <td>RPC Python</td>
   <td></td>
-  <td><a href="https://github.com/deltachat/deltachat-core-rust/tree/master/deltachat-jsonrpc">/deltachat-jsonrpc</a></td>
+  <td><a href="https://github.com/deltachat/deltachat-core-rust/tree/master/deltachat-rpc-client">/deltachat-rpc-client</a></td>
 </tr>
 
 <!-- STANDARDS -->
 
 <tr>
-  <th colspan="3">standards</th>
+  <th colspan="3">Standards</th>
 </tr>
 <tr>
   <td>Autocrypt</td>
   <td><a href="https://autocrypt.org/">autocrypt.org</a></td>
   <td><a href="https://github.com/autocrypt/autocrypt">autocrypt/autocrypt</a></td>
 </tr>
 <tr>
-  <td>verified contacts/groups</td>
-  <td><a href="https://countermitm.readthedocs.io/">countermitm.readthedocs.io</a></td>
+  <td>Verification</td>
+  <td><a href="https://countermitm.readthedocs.io/">countermitm</a></td>
   <td><a href="https://github.com/nextleap-project/countermitm">nextleap-project/countermitm</a></td>
 </tr>
 <tr>
-  <td>others</td>
+  <td>Other</td>
   <td></td>
   <td><a href="https://github.com/deltachat/deltachat-core-rust/blob/master/standards.md">standards.md</a></td>
 </tr>
 
-</table>
+<tr>
+  <th colspan="3">Misc.</th>
```

Comment:
```suggestion
  <th colspan="3">Etcetera</th>
```

Maybe we could use the unabbreviated form, or another word?

Reply:
at least in my world, as a non-native english, "Etcetera" is much more weird as "Misc." (although, "Etc." is known of course :) i also think, ppl will stumble upon "Miscellaneous" more than over "Misc.".

maybe "Various"?

Reply:
> maybe "Various"?

Sure, or 'Elsewhere' or 'On the web'. But not a big deal: I just see the abbreviation dot in the context of the dot separators and it makes me want to remove it.